fs/fsx.py.data/fsx-mm.yaml

```diff
@@ -1,2 +1,13 @@
+url: 'https://github.com/linux-test-project/ltp/archive/master.zip'
+
 thp_page_cache: True
-tmpfs_mount_dir: '/mnt/tmpfs/'
+disk:
+mount_dir: '/mnt/tmpfs'
```

Comment:
can this be only `dir`
similar to other test parameter in yaml

Reply:
sure @Naresh-ibm